docs/EN_US/HPCCSpark/SparkHPCC.xml

```diff
@@ -383,11 +353,10 @@ write.df(df, source = "hpcc",
       <para>We have provided some examples of utilizing a Spark environment.
       The examples provided are dependent on the Spark shell.</para>
 
-      <para>You can refer to the examples in the Github repository: </para>
+      <para>You can refer to the examples in the Github repository:</para>
 
       <para><ulink
-      url="https://github.com/hpcc-systems/Spark-HPCC/tree/master/Examples">https://github.com/hpcc-systems/Spark-HPCC/tree/master/Examples</ulink>
-      </para>
+      url="https://github.com/hpcc-systems/Spark-HPCC/tree/master/Examples">https://github.com/hpcc-systems/Spark-HPCC/tree/master/Examples</ulink></para>
```

Comment:
This needs to be changed to the new HPCC4j URL: https://github.com/hpcc-systems/hpcc4j/tree/master/spark-hpcc/Examples

Reply:
what about the link on line 77-78 ?
https://github.com/hpcc-systems/Spark-HPCC
that seems like it should point to:   https://github.com/hpcc-systems/hpcc4j

Reply:
@g-pan you're correct. However this PR is meant to fix broken links in the docs. 
the link you referenced is still valid (soon to be outdated). https://hpccsystems.atlassian.net/browse/HPCC4J-669 was created for that task of updating it when the time comes. You're welcome to update the link(s) here instead if you prefer.